webapp/app/js/controllers/main.js

```diff
@@ -68,6 +69,16 @@ treeherder.controller('MainCtrl', [
                         thEvents.selectPreviousUnclassifiedFailure
                     );
 
+                }else if(ev.keyCode == 39){
+                    //Highlight next next keys:<right arrow>
```

Comment:
There's an extra 'next' here. The comment can also just say `// Highlight next job, key: right arrow`, and similar wording for previous job, farther below.